com.unity.render-pipelines.universal/Editor/ShaderGraph/Targets/UniversalSpriteLitSubTarget.cs

```diff
@@ -186,6 +187,54 @@ static class SpriteLitPasses
                 // Custom Interpolator Support
                 customInterpolators = CoreCustomInterpDescriptors.Common
             };
+
```

Comment:
There is a new SubTarget in 2d `UniversalSpriteCustomLitSubTarget` which would require this treatment

Reply:
@UnityPaul Is this really new? We haven't merged "master" in a couple of weeks and I can't see the file locally - once it's in I'll add the support

Reply:
yes, it's new, it merged last week. I can update it once this PR lands.